addon/styles/_frost-guage.scss

```diff
@@ -0,0 +1,33 @@
+.frost-guage {
+  position: relative;
+
+  &-text {
```

Comment:
@job13er - avoiding the & is just a reflection of my personal experience around the difference for when I was using the &- pattern and when I wasn't.  I've definitely had moments when I've thought "oh, so there is no class" incorrectly when using &-, but I've never experienced that when being explicit in the SASS files.

On the flipside, the concat in templates has been good, since the vast majority of the time I'm looking _from_ a template _to_ a SASS file, but not the other way around.